bin/run-benchmarks.sh

```diff
@@ -36,9 +37,8 @@ main() {
     mkdir -p "$BLOOP_HOME"
 
     JMH_CMD="$BLOOP_JMH_RUNNER"
-    LOG_FILE="$BLOOP_HOME/$(date +"%F-%H%M%S")-benchmarks-log.txt"
```

Comment:
Can you hoist that value up so that `LOG_FILE` has a default value?

Reply:
What is the value of doing that? We check against the existence of `LOG_FILE` in the trap exit handler.

Reply:
But I don't want the script to crash if I didn't give it a `--logfile`

Reply:
I do, I really wanna make sure that both the listener and this script are on-sync. Do you run this manually?